dotnet/src/Connectors/Connectors.Memory.Postgres/SqlBuilder.cs

```diff
@@ -16,10 +16,10 @@ namespace Microsoft.SemanticKernel.Connectors.Postgres;
 /// <summary>
 /// Provides methods to build SQL commands for managing vector store collections in PostgreSQL.
 /// </summary>
-internal class PostgresVectorStoreCollectionSqlBuilder : IPostgresVectorStoreCollectionSqlBuilder
+internal static class SqlBuilder
```

Comment:
Is the rename here intentional? If we have more than one type with the same name in the solution (e.g. Sqlite, PG, SQL Server all have SqlBuilder), it becomes a bit tough to navigate... Though I agree PostgresVectorStoreCollectionSqlBuilder is needlessly long (PostgresSqlBuilder?)